contributing.md

```diff
@@ -0,0 +1,134 @@
+# Contributing to the Activity Stream  Firefox addon
```

Comment:
Global: addon should be hyphenated (add-on), per the README in this repo, and AMO.